packages/next/package.json

```diff
@@ -60,6 +60,10 @@
     ]
   },
   "dependencies": {
+    "node-sass": "^4.0.0 || ^5.0.0",
+    "sass": "^1.3.0",
+    "fibers": ">= 3.1.0",
```

Comment:
These should be optional peer dependencies, right?

Reply:
Because these are peer dependencies of these plugins, they should become direct next dependencies when we inline these plugins as that's what the inlining operation does.

Do you think there would be benefits to making them optional? Surely that would be a breaking change?